test_db.php

```diff
@@ -0,0 +1,5 @@
+<?php
+include("include\initialize.php");
+$result = dbQuery("SHOW TABLES")->fetchAll();
```

Comment:
I assume this is just for testing since you probably don't want to show your DB tables on the website